tethys_cli/install_commands.py

```diff
@@ -748,24 +757,38 @@ def install_command(args):
                     if has_module(conda_run):
                         conda_config = requirements_config["conda"]
                         install_packages(
-                            conda_config, update_installed=args.update_installed
+                            conda_config,
+                            update_installed=args.update_installed,
                         )
                     else:
-                        write_warning(
-                            "Conda is not installed. Attempting to install conda packages with pip..."
-                        )
-                        try:
-                            call(
-                                [
-                                    "pip",
-                                    "install",
-                                    *requirements_config["conda"]["packages"],
-                                ]
-                            )
-                        except Exception as e:
-                            write_error(
-                                f"Installing conda packages with pip failed with the following exception: {e}"
+                        write_warning("Conda is not installed...")
+                        if not args.quiet:
+                            proceed = input(
+                                "Attempt to install conda packages with pip and continue the installation process: [y/n]"
                             )
+                            while proceed not in ["y", "n", "Y", "N"]:
+                                proceed = input('Please enter either "y" or "n": ')
+                        else:
+                            proceed = "y"
+
+                        if proceed in ["y", "n", "Y", "N"]:
+                            if proceed in ["y", "Y"]:
+                                try:
+                                    call(
+                                        [
+                                            "pip",
+                                            "install",
+                                            *requirements_config["conda"]["packages"],
+                                        ]
+                                    )
+                                except Exception as e:
+                                    write_error(
+                                        f"Installing conda packages with pip failed with the following exception: {e}"
+                                    )
+                            else:
+                                write_msg("\nInstall Command cancelled.")
+                                exit(0)
```

Comment:
I don't think ``if proceed in ["y", "n", "Y", "N"]:`` is necessary b/c of line 769, so this can be simplified:

```suggestion
                        if proceed.lower() in ["y"]:
                            try:
                                call(
                                    [
                                        "pip",
                                        "install",
                                        *requirements_config["conda"]["packages"],
                                    ]
                                )
                            except Exception as e:
                                write_error(
                                    f"Installing conda packages with pip failed with the following exception: {e}"
                                )
                        else:
                            write_msg("\nInstall Command cancelled.")
                            exit(0)
```